pkg/scaler/machineSetScaler.go

```diff
@@ -38,9 +39,14 @@ func (s *machineSetScaler) CanScale(c client.Client, logger logr.Logger) (bool,
 		client.MatchingLabels{"hive.openshift.io/machine-pool": "worker"},
 	}...)
 	if err != nil {
-		logger.Error(err, "failed to get original machinesets")
-		return false, err
+		if apierrors.IsNotFound(err) {
+			logger.Error(err, "failed to get original machinesets")
```

Comment:
```suggestion
			logger.Info(err, "failed to get original machinesets")
```
If it's not an error anymore, I would recommend downgrading this to an info or warning level. I think info makes sense because not having the default machineset isn't a bug/misconfiguration.